### PR TITLE
AFKR-2: Added extension for no vitals taken patients list

### PR DESCRIPTION
**Pull request by rubailly**
To add the translation for the tab, I had to override `locale_en.json` and `locale_fr.json` from the parent repo.

### CODE REVIEW

File: configuration/openmrs/apps/clinical/extension.json [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+    "order": 1,
+    "requiredPrivilege": "app:clinical"
+  },
+  "bahmniClinicalPatientsSearchNoVitalsPatient": {
+    "id": "bahmni.clinical.patients.search.noVitalsPatients",
+    "extensionPointId": "org.bahmni.patient.search",
+    "type": "config",
+    "extensionParams": {
+      "searchHandler":"emrapi.sqlSearch.noVitalsPatients",
+      "translationKey": "MODULE_LABEL_NO_VITALS_KEY",
```
Comment (rbuisson) [inline configuration/openmrs/apps/clinical/extension.json]: `MODULE_LABEL_AWAITING_VITALS` seems a better fit.

File: configuration/openmrs/i18n/clinical/locale_en.json [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+  "MODULE_LABEL_ACTIVE_KEY": "Active",
+  "MODULE_LABEL_PROGRAMS_KEY": "Programs",
+  "MODULE_LABEL_MY_PATIENTS_KEY": "My Patients",
+  "MODULE_LABEL_ACTIVE_PATIENTS_BY_LOCATION": "PatientsByLocation",
+  "MODULE_LABEL_OPD_1_KEY": "OPD-1",
+  "MODULE_LABEL_ALL_KEY": "All",
+  "MODULE_LABEL_EMERGENCY_KEY":"Emergency",
+  "MODULE_LABEL_PRIORITY_KEY":"Priority",
+  "MODULE_LABEL_QUEUE_KEY":"Queue",
+  "MODULE_LABEL_NO_VITALS_KEY": "Vitals not taken",
```
Comment (rbuisson) [inline configuration/openmrs/i18n/clinical/locale_en.json]: label as "Awaiting vitals"

File: configuration/openmrs/i18n/clinical/locale_fr.json [hunk 3]
```diff
@@ -0,0 +1,402 @@
...
+  "MODULE_LABEL_ACTIVE_KEY": "Actifs",
+  "MODULE_LABEL_PROGRAMS_KEY": "Programmes",
+  "MODULE_LABEL_MY_PATIENTS_KEY": "Mes Patients",
+  "MODULE_LABEL_ACTIVE_PATIENTS_BY_LOCATION": "Patients/Emplacement",
+  "MODULE_LABEL_OPD_1_KEY": "OPD-1",
+  "MODULE_LABEL_ALL_KEY": "Tous",
+  "MODULE_LABEL_EMERGENCY_KEY": "Urgents",
+  "MODULE_LABEL_PRIORITY_KEY": "Prioritaires",
+  "MODULE_LABEL_QUEUE_KEY": "Non Urgents",
+  "MODULE_LABEL_NO_VITALS_KEY": "Signes vitaux non pris",
```
Comment (rbuisson) [inline configuration/openmrs/i18n/clinical/locale_fr.json]: Label as "En attente des signes vitaux" (if that's not too long).

**Review by rbuisson — CHANGES_REQUESTED** [review]
Thanks @rubailly , I've left few comments.

File: configuration/openmrs/apps/clinical/extension.json (hunk 1 above)
Comment (rubailly) [reply configuration/openmrs/apps/clinical/extension.json]: [Done](https://github.com/mekomsolutions/bahmni-config-easydoc/pull/7/commits/318c6c514390c51e164fd68b568bb984926f73b2)

File: configuration/openmrs/i18n/clinical/locale_en.json (hunk 2 above)
Comment (rubailly) [reply configuration/openmrs/i18n/clinical/locale_en.json]: [Done](https://github.com/mekomsolutions/bahmni-config-easydoc/pull/7/commits/318c6c514390c51e164fd68b568bb984926f73b2)

File: configuration/openmrs/i18n/clinical/locale_fr.json (hunk 3 above)
Comment (rubailly) [reply configuration/openmrs/i18n/clinical/locale_fr.json]: [Done](https://github.com/mekomsolutions/bahmni-config-easydoc/pull/7/commits/318c6c514390c51e164fd68b568bb984926f73b2)

**Pull request merged**